spec/support/acceptance/helpers.rb

```diff
@@ -0,0 +1,239 @@
+# frozen_string_literal: true
+
+require 'beaker-rspec'
+require 'net/http'
+require 'securerandom'
+require 'yaml'
+
+# Collect global options from the environment.
+LS_VERSION = ENV['LOGSTASH_VERSION'] || '7.17.7'
+PUPPET_VERSION = ENV['PUPPET_VERSION'] || '4.10.7'
+
+PE_VERSION = ENV['BEAKER_PE_VER'] || ENV['PE_VERSION'] || '3.8.3'
+PE_DIR = ENV['BEAKER_PE_DIR']
+
+IS_PRERELEASE = if LS_VERSION =~ %r{(alpha|beta|rc)}
+                  true
+                else
+                  false
+                end
+
+def agent_version_for_puppet_version(puppet_version)
```

Comment:
This should be removed. We don't test on Puppet 4 anymore so if it's still called it's redundant.